warning: in the working copy of 'C:\Users\hhyyl\Downloads\Stronghold-Protocol-v0.2.1\.scratch\audit-025\base-024\package-lock.json', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'C:\Users\hhyyl\Downloads\Stronghold-Protocol-v0.2.1\.scratch\audit-025\base-025\package-lock.json', LF will be replaced by CRLF the next time Git touches it
diff --git "a/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-024\\package-lock.json" "b/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-025\\package-lock.json"
index e70f9e3..fd39792 100644
--- "a/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-024\\package-lock.json"
+++ "b/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-025\\package-lock.json"
@@ -3 +3 @@
-  "version": "0.2.4",
+  "version": "0.2.5",
@@ -9 +9 @@
-      "version": "0.2.4",
+      "version": "0.2.5",
